taskRuby.rb

```diff
@@ -0,0 +1,23 @@
+#Functions
+def ask(question)
+    print question
+    end

```

Comment:
your indentation is a bit off here, as well as spacing.  I usually like to leave at least one blank line between each method.  Adds a bit of clarity to your code